### PR TITLE
 English translation (closes #38) 

### LINKED ISSUE #38: Traduire le site en anglais
Pour les anglophones et les autres, on pourrais traduire le site en anglais.

### PR DESCRIPTION
**Pull request by WeftWiki**

### CODE REVIEW

File: locales.json [hunk 1]
```diff
@@ -117,5 +117,130 @@
...
+        "post_office": "Post office",
+        "pub": "Pub",
+        "restaurant": "Restaurant",
+        "seafood": "Seafood shop",
+	"social_facility": "Social facility",
+        "stationery": "Office supply shop",
+        "supermarket": "Supermarket",
+        "telecommunication": "Telecomunication company office",
+        "tobacco": "Tobacco shop",
+        "wine": "Wine shop"
```
Comment (PanierAvide) [inline locales.json]: Misses a `,` at the end of line

File: locales.json [hunk 2]
```diff
@@ -117,5 +117,130 @@
...
+        "state": {
+          "open": "open",
+          "closed": "closed"
+        }
+      },
+      "state": {
+        "ouvert": "Place open during containment",
+        "ouvert_adapté": "Place open during containment with adapted hours",
+        "partiel": "Place potentially open during containment (to be checked)",
+        "inconnu": "Place whose open status is not known"
```
Comment (PanierAvide) [inline locales.json]: Misses a `,` at the end of line

Comment (WeftWiki) [reply locales.json]: done, thank you

File: locales.json (hunk 1 above)
Comment (WeftWiki) [reply locales.json]: done, thank you

**Pull request merged**